### DISCUSSION TITLE
Is there a way to reduce the size of the second row of a two rows grid?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#6141] [Help] [answered]
I couldn't understand if there is a way to reduce the size of the second row of a grid of two rows.

Example: https://play.tailwindcss.com/0l1gx2l1uK

```html
<div class="grid grid-rows-2 items-center bg-red-200">
  <div class="w-24 h-24 bg-green-600 rounded-full"></div>

  <div class="text-xs">Hello</div>
</div>
```

I need to reduce the height of the `Hello` row.

Is there a way?

![image](https://user-images.githubusercontent.com/41120635/142708506-a8d7b9cc-3cbd-43e0-8793-2b2ab68e0f6c.png)

**Comment by dance2die** [comment, marked as answer]
You can do `grid-rows-1` or  `grid-rows-[1fr]` with JIT mode on to make each row to be flexible.  
At this point, there is no need to use Grid but go with Flexbox with column direction.

```html
<div class="grid grid-rows-[1fr] items-center bg-red-200">
  <div class="w-24 h-24 bg-green-600 rounded-full"></div>

  <div class="text-xs">Hello</div>
</div>
```
